tests/test_surfaces.py

```diff
@@ -102,16 +102,16 @@ def test_from_input_file(self):
     @pytest.mark.unit
     def test_from_near_axis(self):
         """Test constructing approximate QI surface from near axis parameters."""
-        surf = FourierRZToroidalSurface.from_near_axis(10, 4, 0.3, 0.2)
+        surf = FourierRZToroidalSurface.from_qp_model(1, 10, 4, 0.3, 0.2)
         np.testing.assert_allclose(
             surf.R_lmn,
-            np.array([0.075, 0, 1, 0.125, 0, 0.0150853, -0.2, 0.075]),
+            np.array([-0.075, 0, 1, -0.125, 0, 0.030707, -0.2, -0.075]),
```

Comment:
is this something manually calculated or just a regression test where you ran it once, checked the surface, then put that as the asnwer  to test against? if the latter, it should be a regression test, not a unit test